plugins/commands/scanner/src/main/kotlin/ScannerCommand.kt

```diff
@@ -73,22 +69,15 @@ class ScannerCommand : OrtCommand(
     name = "scan",
     help = "Run external license / copyright scanners."
 ) {
-    private val input by mutuallyExclusiveOptions(
+    private val input by
```

Comment:
Formatting / indentation should now also be changed, see e.g. line 82 and following.